### PR TITLE
blockdev_mirror_firewall:omit host dmesg check

### PR DESCRIPTION
**Pull request by aliang123**
Omit driver deprecated warning in host_dmesg.log
Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2173502

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_mirror_firewall.q35: PASS (288.32 s)

**Comment by aliang123** [comment]
@zhencliu @XueqiangWei @maxujun, please help to review, thanks.

File: qemu/tests/cfg/blockdev_mirror_firewall.cfg [hunk 1]
```diff
@@ -31,6 +31,8 @@
     image_name_data1 = data1
     image_name_mirror1 = mirror1
     rebase_mode = unsafe
+    #Fixme if no driver deprecated warning in host_dmesg.log
+    host_dmesg_ignore = yes
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_mirror_firewall.cfg:35]: The same concern as #3623 , it's expected to fail the tc here, when the warning msg issue is fixed, the tc passed. @aliang123  @XueqiangWei

Comment (aliang123) [reply qemu/tests/cfg/blockdev_mirror_firewall.cfg:35]: @zhencliu, warning msg of driver deprecated should be omitted, I think.

Comment (zhencliu) [reply qemu/tests/cfg/blockdev_mirror_firewall.cfg:35]: what about using expected_host_dmesg, e.g. 
expected_host_dmesg = Warning: Deprecated Driver is detected
Then the related warning msgs will be ignored;

In this way, we will not ignore others if sth wrong happens

Comment (aliang123) [reply qemu/tests/cfg/blockdev_mirror_firewall.cfg:35]: @zhencliu, ok, this seems another solution. But once the deprecated driver issue resolved, we also need to update the cfg again.

Comment (zhencliu) [reply qemu/tests/cfg/blockdev_mirror_firewall.cfg]: Also, I'm curious why we didn't hit this for other testings, this is a post check, does this testing introduce the deprecated driver only? @aliang123

Comment (aliang123) [reply qemu/tests/cfg/blockdev_mirror_firewall.cfg]: (1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_mirror_firewall.q35: PASS (289.99 s)

Comment (aliang123) [reply qemu/tests/cfg/blockdev_mirror_firewall.cfg]: > Also, I'm curious why we didn't hit this for other testings, this is a post check, does this testing introduce the deprecated driver only? @aliang123
@zhencliu Yes, zhenchao, only this case can hit this issue, other cases work ok.
After checking the test result recoreds, we can see that sometimes we can hit this issue, and sometimes not.
So only define "expected_host_dmesg = Warning: Deprecated Driver is detected" here is not aviable now.
As it's just some warning msg, we can omit them.
May we need to modify them to "expected_host_dmesg = Warning: Deprecated Driver is detected | ".

**Comment by aliang123** [comment]
Test the patch with a not-hit driver deprecated version, case also pass.
(1/1) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_mirror_firewall.q35: PASS (222.04 s)

**Comment by aliang123** [comment]
@zhencliu, have updated the test result, please help to check, thanks.

**Comment by maxujun** [comment]
(1/1) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.ppc64le.io-github-autotest-qemu.blockdev_mirror_firewall: PASS (410.53 s)
LGTM,Acked

**Comment by zhencliu** [comment]
LGTM
hi @XueqiangWei , would you please make a double check? Thanks.

**Comment by XueqiangWei** [comment]
(1/1) Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_mirror_firewall.q35: PASS (229.03 s)

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by: xuwei@redhat.com

**Review by zhencliu — APPROVED** [review]

**Pull request merged**